#################################
#
# Gesture Detection
#
#################################

std_msgs/String Gesture_name
int32 Gesture_id

geometry_msgs/Pose gesture_position


